samples/hello_fpga.c

```diff
@@ -363,7 +363,7 @@ int main(int argc, char *argv[])
 	ON_ERR_GOTO(res1, out_free_output, "writing CSR_CFG");
 
 	/* Wait for test completion */
-	timeout = 30000;
+	timeout = 130000000;
```

Comment:
Without adding the timeout, the test still fails, right? So, basically it is a timing issue.

Reply:
Basically extending the timeout is the key to fix the NLB failure issue. We will look deeper into the RTL design with Michael Adler to see whether there is a hw issue.